### PR TITLE
MicrosoftCorporation.MicrosoftBuildTools2015 version 14.0.23107.10

### PR DESCRIPTION
**Pull request by Scavy**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/96844)

### CODE REVIEW

File: manifests/m/MicrosoftCorporation/MicrosoftBuildTools2015/14.0.23107.10/MicrosoftCorporation.MicrosoftBuildTools2015.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.MicrosoftBuildTools2015
```
Comment (stephengillie) [inline manifests/m/MicrosoftCorporation/MicrosoftBuildTools2015/14.0.23107.10/MicrosoftCorporation.MicrosoftBuildTools2015.installer.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.BuildTools2015
```

Comment (stephengillie) [reply manifests/m/MicrosoftCorporation/MicrosoftBuildTools2015/14.0.23107.10/MicrosoftCorporation.MicrosoftBuildTools2015.installer.yaml:4]: Along with this suggestion, the file paths and names will need to be updated to match.

File: manifests/m/MicrosoftCorporation/MicrosoftBuildTools2015/14.0.23107.10/MicrosoftCorporation.MicrosoftBuildTools2015.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.MicrosoftBuildTools2015
```
Comment (stephengillie) [inline manifests/m/MicrosoftCorporation/MicrosoftBuildTools2015/14.0.23107.10/MicrosoftCorporation.MicrosoftBuildTools2015.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.BuildTools2015
```

Comment (stephengillie) [reply manifests/m/MicrosoftCorporation/MicrosoftBuildTools2015/14.0.23107.10/MicrosoftCorporation.MicrosoftBuildTools2015.locale.en-US.yaml:4]: Along with this suggestion, the file paths and names will need to be updated to match.

File: manifests/m/MicrosoftCorporation/MicrosoftBuildTools2015/14.0.23107.10/MicrosoftCorporation.MicrosoftBuildTools2015.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.MicrosoftBuildTools2015
```
Comment (stephengillie) [inline manifests/m/MicrosoftCorporation/MicrosoftBuildTools2015/14.0.23107.10/MicrosoftCorporation.MicrosoftBuildTools2015.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.BuildTools2015
```

Comment (stephengillie) [reply manifests/m/MicrosoftCorporation/MicrosoftBuildTools2015/14.0.23107.10/MicrosoftCorporation.MicrosoftBuildTools2015.yaml:4]: Along with this suggestion, the file paths and names will need to be updated to match.

File: manifests/m/MicrosoftCorporation/MicrosoftBuildTools2015/14.0.23107.10/MicrosoftCorporation.MicrosoftBuildTools2015.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.MicrosoftBuildTools2015
+PackageVersion: 14.0.23107.10
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PackageName: Microsoft Build Tools 2015
```
Comment (stephengillie) [inline manifests/m/MicrosoftCorporation/MicrosoftBuildTools2015/14.0.23107.10/MicrosoftCorporation.MicrosoftBuildTools2015.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.microsoft.com/
PrivacyUrl:   https://privacy.microsoft.com/en-us
PackageName: Microsoft Build Tools 2015
PackageUrl: https://www.microsoft.com/en-US/download/details.aspx?id=48159
```

**Comment by Scavy** [comment]
Are there anything wrong with this package? It seems to be stuck somehow. Let me know if there is anything I need to do?

**Review by stephengillie — APPROVED** [review]

**Pull request merged**